syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -1,14 +1,17 @@
+from typing import Dict
+
 from operator import add, sub, mul
 import torch
 from syft.generic.frameworks.hook import hook_args
 import syft
 from syft.generic.abstract.tensor import AbstractTensor
+from syft.workers.abstract import AbstractWorker
 from syft.frameworks.torch.mpc.przs import PRZS, gen_alpha_3of3
 
 
 class ReplicatedSharingTensor(AbstractTensor):
-    def __init__(self, plain_text=None, players=None, ring_size=None, owner=None):
-        super().__init__(owner=owner)
+    def __init__(self, plain_text=None, players=None, ring_size=None, owner=None, id=None):
+        super().__init__(owner=owner, id=id)
```

Comment:
don't forget the tag & description params (like for AST)

Reply:
Done